Dart/src/com/jetbrains/lang/dart/resolve/DartResolver.java

```diff
@@ -129,6 +123,16 @@ else if (leftReference != null) {
   }
 
   @Nullable
+  public PluginNavigationRegion findRegion(final PsiFile refPsiFile, final int refOffset, final int refLength) {
```

Comment:
Done.
IDEA did not make a proposal for this.
I used to use it to know when a method can be made static :-)